packages/strapi-connector-bookshelf/lib/buildQuery.js

```diff
@@ -21,8 +21,18 @@ const buildQuery = ({ model, filters }) => qb => {
   const joinsTree = buildJoinsAndFilter(qb, model, filters);
 
   if (_.has(filters, 'sort')) {
-    const clauses = filters.sort.map(buildSortClauseFromTree(joinsTree));
-    qb.orderBy(reject(isEmpty, clauses));
+    const clauses = filters.sort.map(buildSortClauseFromTree(joinsTree)).filter(c => !isEmpty(c));
+    const aliasedClauses = clauses.map(c => ({
+      order: c.order,
+      column: c.column.includes('.') ? c.tmpColumnName : c.column,
```

Comment:
I did it but still need to filter somewhere to avoid renaming the columns of the main table, otherwise those columns would be duplicated and be present in the query result